substrate/client/chain-spec/Cargo.toml

```diff
@@ -43,3 +43,8 @@ substrate-test-runtime = { path = "../../test-utils/runtime" }
 sp-keyring = { path = "../../primitives/keyring" }
 sp-application-crypto = { default-features = false, path = "../../primitives/application-crypto", features = ["serde"] }
 sp-consensus-babe = { default-features = false, path = "../../primitives/consensus/babe", features = ["serde"] }
+
+[features]
+clap = [
+    "dep:clap"
+]
```

Comment:
If I recall correctly this may not be needed (should be added implicitly).

Reply:
You are right. Thanks!